modules/extensions-spring-support/src/main/java/org/mule/runtime/module/extension/internal/config/dsl/ExtensionDefinitionParser.java

```diff
@@ -932,7 +930,7 @@ protected List<ParameterModel> getFlatParameters(List<ParameterGroupModel> inlin
   }
 
   protected Optional<String> getInfrastructureParameterName(MetadataType fieldType) {
-    return getId(fieldType).map(infrastructureParameterMap::get);
+    return InfrastructureTypeUtils.getInfrastructureParameterName(fieldType);
```

Comment:
static import

Reply:
Not possible, same name as the function calling it.